src/transform.ts

```diff
@@ -184,31 +217,63 @@ export function processCss(src: string, filename: string): TransformedSource {
 // https://jestjs.io/docs/code-transformation#writing-custom-transformers
 // As can be seen, only process or processAsync is mandatory to implement)
 
-// We can use that if we opt-in to ESM. But it's not common use case right now (2022)
-// So our approach is making CSS Modules a "CSS-in-JS" solution.
-// CSS Modules will be compiled at run time then inject to the document.head
-// One notable note is that `postcss-modules` is an async postcss plugin
-// We have to use Singleton design pattern to make it works asynchronously.
-function processCSSModules(src: string, filename: string): TransformedSource {
-  return {
-    code: `const postcss = require('postcss');
-const PostcssModulesPlugin = require('postcss-modules');
-const cssSrc = ${JSON.stringify(src)};
-
-class Token {
-  set(json) {
-    Object.keys(json).forEach((key) => {
-      this[key] = json[key];
-    });
+// Convert from
+//cssModulesExportedTokens||| {"scssModule":"_scssModule_ujx1w_1"}
+// ---
+// css||| ._scssModule_ujx1w_1 {
+//   color: #ff0000;
+// }
+// ---
+// To this
+// {"scssModule":"_scssModule_ujx1w_1", "css":"._scssModule_ujx1w_1 { color: #ff0000 }"}
```

Comment:
I think this should be `{ "cssModulesExportedTokens": { "scssModule":"_scssModule_ujx1w_1"}, "css":"._scssModule_ujx1w_1 { color: #ff0000 }"}` as the return value

Reply:
@ntt261298 Thanks. I pushed the change.